lib/src/Uncertainty/StatTests/HypothesisTest.cxx

```diff
@@ -330,5 +329,18 @@ HypothesisTest::TestResultCollection HypothesisTest::FullSpearman(const Sample &
   return PartialSpearman(firstSample, secondSample, selection, level);
 }
 
+TestResult HypothesisTest::LikelihoodRatioTest(const Scalar model0LogLikelihood,
+                                    const Scalar model1LogLikelihood,
+                                    const Scalar level)
+{
+  const Scalar dp = 2.0 * (model1LogLikelihood - model0LogLikelihood);
+  const Scalar cAlpha = ChiSquare(1.0).computeQuantile(level, true)[0];
```

Comment:
`const Scalar cAlpha = 2.0 * DistFunc::qGamma(0.5, level, true);` to avoid the costly construction of a ChiSquare(1.0) distribution